Changes

```diff
@@ -31,6 +31,7 @@ Verilator 5.023 devel
 * Fix DFG removing forceable signals (#4942). [Geza Lore]
 * Fix null characters in shortened identifiers (#4946). [Abdul Hameed]
 * Fix assignment of null into struct member (#4952).
+* Fix unpacked structure upper bit cleaning (#4978).
```

Comment:
Just a note for next time you don't need to update Changes, as this can cause merge trouble, I'll do it near release time.

Reply:
I see, alright, I'll keep that in mind. Thanks! And sorry for the troubles 🙏